packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/src/models.ts

```diff
@@ -129,7 +133,11 @@ export interface SharkParent extends FishParent {
 
 export interface Sawshark extends SharkParent {
   /** Value may contain base64 encoded characters */
-  picture?: string;
+  picture?:
+    | string
+    | Uint8Array
+    | ReadableStream<Uint8Array>
+    | NodeJS.ReadableStream;
```

Comment:
@joheredi This would be fine because the new type is compatible with old `string` one.